Added documentation on distribution mappings usage in jfrog file spec (#57)
jfrog/documentation: jfrog-applications/jfrog-cli/cli-for-jfrog-distribution.md

jf ds rbc myApp 1.0.0 "my-local-repo/zips/(*).zip" --target "my-target-repo/target-zips/{1}-target.zip"
```

#### Example 7

> Create a Release Bundle and Apply "pathMapping" to Artifact Paths After Distributing the Release Bundle:

Reviewer: ```suggestion
This example creates a release bundle and applies "pathMapping" to the artifact paths after distributing the release bundle.
```

All occurrences of the "a1.in" file are fetched and mapped to the "froggy" repository at the edges.

1. Fetch all artifacts retrieved by the AQL query.
2. Create the release bundle with the artifacts and apply the path mappings at the edges after distribution.

The "pathMapping" option is provided, allowing users to control the destination of the release bundle artifacts at the edges.

To learn more, visit the [Create Release Bundle v1 Version documentation](https://jfrog.com/help/r/jfrog-rest-apis/create-release-bundle-v1-version).

*Note*: The "target" option is designed to work for most use cases. The "pathMapping" option is intended for specific use cases, such as including a list.manifest.json file inside the release bundle.

In that scenario, the distribution server dynamically includes all the manifest.json and their layers and assigns the given path mapping, whereas "target" doesn't achieve this.
```
jf ds rbc --spec=/path/to/rb-spec.json myApp 1.0.0
```
Spec file content:

> ```

Reviewer: ```suggestion
```json
```

{
  "files": [
    {
      "aql": {
        "items.find": {
          "repo": "my-local-repo",

>           "$or": [
>             {
>               "$and": [
>                 {
>                   "path": {
>                     "$match": "."
>                   },
>                   "name": {
>                     "$match": "a1.in"
>                   }
>                 }
>               ]
>             },
>             {
>               "$and": [
>                 {
>                   "path": {
>                     "$match": "*"
>                   },
>                   "name": {
>                     "$match": "a1.in"
>                   }
>                 }
>               ]
>             }
>           ]

Reviewer: Query can be simplified:
```suggestion
          "$and": [
            {
              "name": {
                "$match": "a1.in"
              }
            },
            {
              "$or": [
                {
                  "path": {
                    "$match": "."
                  }
                },
                {
                  "path": {
                    "$match": "*"
                  }
                }
              ]
            }
          ]
```
